roles/edpm_frr/meta/argument_specs.yml

```diff
@@ -143,7 +143,7 @@ argument_specs:
         description: ''
         type: str
       edpm_frr_hostname:
-        default: '{{ ansible_facts[''hostname''] | default("")}}'
+        default: '{{ ansible_facts[''hostname''] }}'
```

Comment:
This is just to see if we can simulate the failure using molecule. Assuming this does indeed fail in the same way outlined in this bug: https://issues.redhat.com/browse/OSPRH-10338

Then we can revert this specific part of the change

Reply:
Indeed, this did manage to reproduce the failure:
https://softwarefactory-project.io/zuul/t/rdoproject.org/build/745dcfd57bf548a1a43d61aa7d899476